recipes/targetscan/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+mkdir -p $PREFIX/bin
+cp targetscan_70.pl $PREFIX/bin
```

Comment:
I see a bunch of other Perl scripts at https://www.targetscan.org/cgi-bin/targetscan/data_download.vert80.cgi you could have included. 

But I do think the license is required.

Reply:
I'll need to create a PR for the perl module Bio::[TreeIO](https://metacpan.org/pod/Bio::TreeIO) for the subsequent targetscan scripts to work, I can try to 'paint by numbers' using the multiple examples of perl modules in the recipes directory.

Regarding the license file, I honestly cannot find one. I've searched their 8 publications to no avail, all I can glean is this info from their executable: `Copyright (c) The Whitehead Institute of Biomedical Research`. 

I never got the sense Targetscan is proprietary, it is routinely used in papers that use miRNA prediction. Perhaps this is a result of the tool being developed in 2005. Alas... let me know your gut feeling, I can open a discussion on nf-core.

Reply:
I'm not one of the core Bioconda people, but IMO I don't think this is going to fly without an appropriate license. You should ask on the Bioconda gitter in case I'm wrong (this isn't really an nf-core issue).